lock_test.go

```diff
@@ -134,7 +134,7 @@ var staleLockTests = []struct {
 		timestamp:        time.Now(),
 		stale:            true,
 		staleOnOtherHost: false,
-		pid:              os.Getpid() + 500,
+		pid:              os.Getpid() + 500000,
```

Comment:
The correction of a merge error of the following commit:
https://github.com/restic/restic/commit/3804bc7493711782d022164f9dc95b8e1c79a700

I've also stumbled upon this in https://github.com/restic/restic/pull/259#discussion_r37098341